test/others/libcriu/test_feature_check.c

```diff
@@ -0,0 +1,65 @@
+#include "criu.h"
+#include <fcntl.h>
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+#include "lib.h"
+
+int main(int argc, char **argv)
+{
+	int ret;
+	char *env;
+	int mem_track = 0;
+	int lazy_pages = 0;
+	int pidfd_store = 0;
+	struct criu_feature_check features = {
+		.mem_track = true,
+		.lazy_pages = true,
+		.pidfd_store = true,
+	};
+
+	printf("--- Start feature check ---\n");
+	criu_init_opts();
+	criu_set_service_binary(argv[1]);
+
+	env = getenv("CRIU_FEATURE_MEM_TRACK");
+	if (env) {
+		mem_track = atoi(env);
```

Comment:
Do we really need atoi here? Why not mem_track = true?

Reply:
Yes, not necessary. Removed.